_data/twenty_19/faqs.yml

```diff
@@ -1,11 +1,5 @@
 - question: "How much does participation cost?"
-  answer: "There is no participation fee. Lunches and coffee breaks will be
-  covered."
-
-- question: "Is there anything else for free?"
-  answer: "We can offer a limited number of rooms during the school (four
-  nights). For more details, see the <a href=\"../register\">registration
-  page</a> (<em>the registration deadline was August 31</em>)."
+  answer: "There is no participation fee."
```

Comment:
@renefritze It looks like the wrong year was updated.

Reply:
D'oh. Fixed now!